views/models/ChartFiltersModel/index.js

```diff
@@ -0,0 +1,33 @@
+const ChartFiltersModel = {
+  filtersToFormValues: (filters) => {
+    const filterKeys = Object.keys(filters)
+
+    return filterKeys.reduce((formValues, filterKey) => {
+      formValues[filterKey] = Object.values(filters[filterKey])
+        .filter(({ value }) => value === 1)
+        .map(({ label }) => label)
```

Comment:
why are we using the labels and not the values for the form values?

Reply:
Good question, 
 
With react select we were able to pass in an object with `{label: 'some label', value: 1`} but the MUI select component is unable to process an array of objects and can only handle an array of strings. The goal of this function is to look at the filters with the value of 1 and pass that to the MUI select.

We use 1 and 0 to let us know that a filter is active or inactive, and this is because the `qs` library stringifies everything, including booleans, using 1 and 0 is simpler for on an off, otherwise, we'd have to process a string with the value of true or false.

Long winded answer hope that helps.